core/src/com/unciv/logic/map/MapSize.kt

```diff
@@ -50,23 +50,17 @@ class MapSize private constructor(
         val techCostMultiplier: Float = 1f,
         val techCostPerCityModifier: Float = 0.05f,
         val policyCostPerCityModifier: Float = 0.1f,
+        val unHappinesPerCity: Float = 3f,
```

Comment:
This is 100% a **ruleset issue** and not something we should determine statically for all rulesets
Sounds like a different PR, plus we'll need to think about how exactly to do this - if a player has a custom map size, then what?

Reply:
The techCostPercityModifier and policyCostPerCityModifier are also stored here. I haven't noticed those bugging with custom map sizes.

Reply:
I agree, those too should be rulest-changable

Reply:
You mean those should be added as mod constants for each of the map sizes? That's 3*6 =18 new mod constants.

Reply:
No, I'm saying this should be a **global unique** dependent on map size via conditional
Again, out of scope here. Let's remove this from this PR, turn this PR into just about changing the map sizes, and then we can discuss unhappiness per city per map size in another place.

When you try to add too many things in one PR things can get conflated, which makes the whole thing drag on way longer than it should. One thing at a time

Reply:
I don't really care about the predefined map sizes, this PR is to move the tresholds for science/culture cost modifiers :(

Reply:
It also possible to set the science/culture/happiness cost independently of the predefined map sizes, but I figured such would create more complications, not less.

Reply:
Here's the poll results, option B (keep it the same) got by far the least votes (12%): https://discord.com/channels/586194543280390151/618491418859798539/1350536363547689122

Reply:
> Let's remove this from this PR, turn this PR into just about changing the map sizes

@yairm210 should I set the science/culture/happiness modifier thresholds independently of the predefined map sizes? That way the map sizes can stay the same as before, I just thought it would create unneeded complexity.

Reply:
If we have multiple things that can change by map size, then they obviously need to be moddable.
The question is how. 
Option 1, is to extract all map sizes into a new json file, and then each one can apply certain global uniques
Option 2, is to have a conditional for map sizes
I'm not sure which is better, I'm leaning towards the former...

Reply:
'per city' sounds like an existing countable